src/coreComponents/constitutive/contact/ContactBase.hpp

```diff
@@ -116,10 +117,14 @@ class ContactBaseUpdates
    * @return the updated traction
    */
   GEOSX_HOST_DEVICE
-  inline
+  GEOSX_FORCE_INLINE
   virtual void addPressureToTraction( real64 const & pressure,
```

Comment:
Not sure why we don't see the same exact issue here.. since it is literally the same pattern that caused the bug noted above, yay cce.